enzy_htp/preparation/cluster.py

```diff
@@ -0,0 +1,77 @@
+"""
+"""
+from pathlib import Path
+import pandas as pd
+
+from enzy_htp import interface, config
+
+import enzy_htp.chemical as chem
+
+from enzy_htp.core import file_system as fs
+
+
+def create_cluster(fname: str, sele_str: str, outfile: str = None, cap_strategy: str = 'H', work_dir: str = None, session=None) -> str:
```

Comment:
Same as comments in align_ligand.py.
- /structure objs as IO (you can refer to the use in mutation module)

Reply:
Oh I saw you have it in the PyMolInterface. Which version is the final one? Is this function waiting for update?

Reply:
The one in `PyMolInterface()` is the correct one to use. Let me remove this.